Manifest.toml

```diff
@@ -1,219 +0,0 @@
-# This file is machine-generated - editing it directly is not advised
```

Comment:
Perhaps your earlier problem was checking `Manifest.toml` into the repository? For packages you need only `Project.toml`.

Reply:
Oh, didn't know that 😅